### PR TITLE
SASS-7950 - UK Rent a Room - Allowances: Create 'Other capital allowa…

### PR DESCRIPTION
**Pull request by modivilioglu**
…nces' page

### CODE REVIEW

File: app/controllers/OtherCapitalAllowancesController.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import pages.OtherCapitalAllowancesPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.OtherCapitalAllowancesView
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class OtherCapitalAllowancesController @Inject() (
```
Comment (vivekhmrc) [inline app/controllers/OtherCapitalAllowancesController.scala:33]: Maybe rename this to `RaROtherCapitalAllowancesController`

File: app/pages/OtherCapitalAllowancesPage.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object OtherCapitalAllowancesPage extends QuestionPage[BigDecimal] {
+
+  override def path: JsPath = JsPath \ toString
```
Comment (vivekhmrc) [inline app/pages/OtherCapitalAllowancesPage.scala:23]: In PageConstants we need `val rentARoomAllowances: String = "rentARoomAllowances"` and use it here

File: app/viewmodels/checkAnswers/OtherCapitalAllowancesSummary.scala [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object OtherCapitalAllowancesSummary {
+
+  def row(taxYear: Int, answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
+    answers.get(OtherCapitalAllowancesPage).map { answer =>
+      SummaryListRowViewModel(
+        key = "otherCapitalAllowances.checkYourAnswersLabel",
```
Comment (vivekhmrc) [inline app/viewmodels/checkAnswers/OtherCapitalAllowancesSummary.scala:32]: You are using `ukRentARoom.jointlyLet.checkYourAnswersLabel` at other places. Can this be the same `ukRentARoom.otherCapitalAllowances.checkYourAnswersLabel`

Comment (modivilioglu) [reply app/viewmodels/checkAnswers/OtherCapitalAllowancesSummary.scala:32]: sure, done.

File: app/pages/OtherCapitalAllowancesPage.scala (hunk 2 above)
Comment (modivilioglu) [reply app/pages/OtherCapitalAllowancesPage.scala:23]: Sure, done.

**Review by vivekhmrc — APPROVED** [review]

**Review by vivekhmrc — APPROVED** [review]

**Pull request merged**